metrics/counter_float64.go

```diff
@@ -115,39 +116,52 @@ func (NilCounterFloat64) Snapshot() CounterFloat64 { return NilCounterFloat64{}
 // StandardCounterFloat64 is the standard implementation of a CounterFloat64 and uses the
 // sync.Mutex package to manage a single float64 value.
 type StandardCounterFloat64 struct {
-	mutex sync.Mutex
-	count float64
+	floatBits uint64
```

Comment:
Why not use the new `atomic.Uint64` instead?

Reply:
Because it would result in one extra & operation to pass it to helper functions.

Reply:
For the sake of clarity and adhering to what nowadays is the go-lang recommended practice, please switch to use `atomic.Uint64` instead.

Reply:
Unless I am missing something the overhead can't be that bad (?)